pkgs/applications/misc/gpodder2go/default.nix

```
@@ -0,0 +1,25 @@
+{ buildGoModule, fetchFromGitHub, lib, }:
+buildGoModule rec {
+  pname = "gpodder2go";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "oxtyped";
+    repo = "gpodder2go";
+    rev = "v${version}";
+    hash = "sha256-DLUVANrePlnzEGmyjmrtQbus8zjPytBJUIg2MSqD8go=";
+  };
+
+  checkPhase = false;
```

```suggestion
  doCheck = false;
```

also just to confirm, why are we skipping tests?

---

> also just to confirm, why are we skipping tests?

They just don't pass while running in nix for some reason.